crates/store/re_types/definitions/rerun/blueprint/datatypes/component_column_selector.fbs

```diff
@@ -14,6 +13,10 @@ table ComponentColumnSelector (
   entity_path: rerun.datatypes.EntityPath (order: 100);
 
   /// The name of the component.
+  ///
+  /// If the component does not have an associated `ArchetypeName`, this is just a plain field name. Otherwise,
+  /// this is a combination of `<ArchetypeName>:<ArchetypeFieldName>`. An example for this would be
+  /// `Points3D:positions`, for the `positions` field in the `Points3D` archetype.
   // NOTE: this field cannot be named `component_name`, because it would then clash with
   // `ComponentMixin.component_name()` in the Python API.
   component: rerun.datatypes.Utf8 (order: 200);
```

Comment:
As per our discussion with Jeremy this is true for Rerun builtins, but more general this is just the column name.

Would it be better to write it like this:


```suggestion
  /// The name of the component.
  ///
  /// This acts as the column name in the context of a given `entity_path`
  /// By convention, for all Rerun typtes this is a combination of `<ArchetypeName>:<ArchetypeFieldName>`.
  /// (if there's no `ArchetypeName` it is just the field name).
  /// An example for this would be `Points3D:positions`, for the `positions` field in the `Points3D` archetype.
  // NOTE: this field cannot be named `component_name`, because it would then clash with
  // `ComponentMixin.component_name()` in the Python API.
  component: rerun.datatypes.Utf8 (order: 200);
```